### DISCUSSION TITLE
Development setup- backend stuck [May 31]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2281] [Discord Threads]
> 🙋‍♂️ **ryzenway**  
## My nocodb backend is stuck at below. It is not moving forward. I followed proper steps for deploying on local. In case someone has faced this please let me know  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/980810590450761768/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### I tried v0.91.1 from develop, was able to start just fine. Please ensure port 8080 is not in use.  
#  
> 👨‍🦲 **dstala**  
#### If it fails still, follow steps here   
#### https://github.com/nocodb/nocodb#development-setup   
####    
#### ensure we have built SDK as well. build SDK step is missing in noco-docs (https://docs.nocodb.com/getting-started/installation#development-setup), will update it.  
#  
> 🙋‍♂️ **ryzenway**  
#### Thanks will check  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/980810591117668382)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
